Colours_And_Fonts_style_v1.css

```diff
@@ -62,12 +127,13 @@ p {
 /* CSS for styling my Navigation/Menu */
 .nav {
   box-sizing: border-box;
-  background-color: #2c7da0; /*Blue*/
+  background-color: #3750A5; /*Blue*/
   grid-column-start: 1;
   grid-column-end: span 2;
   width: 1990px;
-  height: 50px;
+  height: 100px;

```

Comment:
I made the height of the nav from 50px to 100px so I can make the font size bigger.